raiden/tests/utils/transfer.py

```diff
@@ -1077,3 +1077,29 @@ def block_timeout_for_transfer_by_secrethash(
 
 def calculate_fee_for_amount(amount: int) -> int:
     return round((amount * INTERNAL_ROUTING_DEFAULT_FEE_PERC) * (1 + DEFAULT_MEDIATION_FEE_MARGIN))
+
+
+def drain_eth_account(raiden: RaidenService):
```

Comment:
There is already a test utility function serving the same purpose here: https://github.com/raiden-network/raiden/blob/5c1ce4220e74d4e84a2522896a91aa7b7c955c3e/raiden/tests/utils/client.py#L8

Please use that instead.